src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -2217,8 +2217,8 @@ private ToolStripItem GetNextItemHorizontal(ToolStripItem start, bool forward)
                 return null;
             }
 
-            Debug.WriteLineIf(s_selectionDebug.TraceVerbose && (current != -1), "[SelectDBG GetNextToolStripItem] Last selected item was " + ((current != -1) ? DisplayedItems[current].Text : ""));
-            Debug.WriteLineIf(s_selectionDebug.TraceVerbose && (current == -1), "[SelectDBG GetNextToolStripItem] Last selected item was null");
+            Debug.WriteLineIf(s_selectionDebug!.TraceVerbose && (current != -1), "[SelectDBG GetNextToolStripItem] Last selected item was " + ((current != -1) ? DisplayedItems[current].Text : ""));
```

Comment:
We are trying to use string interpolation throughout our codebase instead now
```suggestion
            Debug.WriteLineIf(s_selectionDebug!.TraceVerbose && (current != -1), $"[SelectDBG GetNextToolStripItem] Last selected item was {((current != -1) ? DisplayedItems[current].Text : "")}");
```